service/src/main/java/bio/terra/cbas/initialization/InstanceInitializer.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.cbas.initialization;
+
+import org.jetbrains.annotations.NotNull;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class InstanceInitializer implements ApplicationListener<ContextRefreshedEvent> {
+
+  private final BackfillOriginalWorkspaceIdBean backfillOriginalWorkspaceIdBean;
+
+  public InstanceInitializer(BackfillOriginalWorkspaceIdBean backfillOriginalWorkspaceIdBean) {
+    this.backfillOriginalWorkspaceIdBean = backfillOriginalWorkspaceIdBean;
```

Comment:
Maybe I was mistaken above (about not calling this a bean).... it certainly seems like it's being autowired here. But it's not marked as something that can be considered a bean 🤔